lib/driver.js

```diff
@@ -259,10 +259,19 @@ class XCUITestDriver extends BaseDriver {
     log.info(`${this.opts.bundleId} is in foreground`);
   }
 
-  async startWdaSession (app, bundleId) {
+  async startWdaSession (app, bundleId, processArguments) {
+    let appArguments = [];
```

Comment:
Thanks. I will do that style change with argsv and env. I know about arguments, in fact it didnt allow me to use that as variable but FB expect like this app.launchArguments = (NSArray *)requirements[@"arguments"] ?: @[];
So i ended up using different variable name but still requests needs to be send with arguments keyword